solutions/USACO feb19-the-great-revegetation.mdx

```diff
@@ -0,0 +1,116 @@
+---
+id: usaco-920
+source: USACO Silver 2019 February
+title: The Great Revegetation
+author: Sofia Yang
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_revegetate_silver_feb19.html)
+
+
+<LanguageSection>
+
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.ArrayList;
+import java.util.StringTokenizer;
+
+public class revegetate {
+	public static int[] type;
+	public static boolean impossible;
+	public static ArrayList<Integer>[] same;
+	public static ArrayList<Integer>[] diff;
+
+	public static void main(String[] args) throws IOException {
+		BufferedReader br = new BufferedReader(new FileReader("revegetate.in"));
+		PrintWriter pw = new PrintWriter(new FileWriter("revegetate.out"));
+		StringTokenizer st = new StringTokenizer(br.readLine());
+		int N = Integer.parseInt(st.nextToken());
+		int M = Integer.parseInt(st.nextToken());
+
+		type = new int[N + 1];
+		same = new ArrayList[N + 1];
+		diff = new ArrayList[N + 1];
+
+		//Initialize the adjacency lists.
+		for (int i = 0; i < N + 1; i++) {
```

Comment:
```suggestion
		for (int i = 0; i <= N; i++) {
```
inconsistent formatting (used `for (int i = 0; i <= N; i++)` below) change this or the one below